### ISSUE TITLE
wineserver not found!

### ISSUE THREAD

**Issue by EmmanuelD69** [rockerbacon/lutris-skyrimse-installers#91]
Hello Everyone,
This is now about 1 week i am blocked on the install of MO2.
I reinstalled Linux not too long ago and so though i would try to install for relaxing Skyrim with some moding action.
Problem is i am unable to successfully install MO2.

The messages windows i get from Lutris during the install process are:

Lutris Error window message:
Error while installing winetricks, please run Lutris from a terminal and check the logs

Lutris installation window message:
I can't move /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist

The Shell transcript is:
2020-06-28 22:10:11,360: Running Lutris 0.5.6
2020-06-28 22:10:11,361: Using NVIDIA drivers 440.100 for x86_64
2020-06-28 22:10:11,361: GPU: GeForce GTX 1070
2020-06-28 22:10:11,361: GPU: 10DE:1B81 1462:3302 using nvidia drivers
2020-06-28 22:10:11,377: Vulkan is supported
2020-06-28 22:10:11,378: Updating DXVK versions
2020-06-28 22:10:11,489: Connected to lutris.net as dodgeman
2020-06-28 22:10:35,108: Running Lutris 0.5.6
2020-06-28 22:10:35,109: Using NVIDIA drivers 440.100 for x86_64
2020-06-28 22:10:35,109: GPU: GeForce GTX 1070
2020-06-28 22:10:35,109: GPU: 10DE:1B81 1462:3302 using nvidia drivers
2020-06-28 22:10:35,125: Vulkan is supported
2020-06-28 22:12:08,476: Downloading file 1 of 16
2020-06-28 22:12:09,488: Downloading file 2 of 16
2020-06-28 22:12:10,495: Downloading file 3 of 16
2020-06-28 22:12:11,501: Downloading file 4 of 16
2020-06-28 22:12:12,506: Downloading file 5 of 16
2020-06-28 22:12:13,512: Downloading file 6 of 16
2020-06-28 22:12:14,519: Downloading file 7 of 16
2020-06-28 22:12:15,525: Downloading file 8 of 16
2020-06-28 22:12:18,035: Downloading file 9 of 16
2020-06-28 22:12:19,041: Downloading file 10 of 16
2020-06-28 22:12:20,045: Downloading file 11 of 16
2020-06-28 22:12:21,051: Downloading file 12 of 16
2020-06-28 22:12:22,558: Downloading file 13 of 16
2020-06-28 22:12:23,563: Downloading file 14 of 16
2020-06-28 22:12:24,566: Downloading file 15 of 16
2020-06-28 22:12:25,569: Downloading file 16 of 16

(lutris:5705): Gtk-CRITICAL **: 22:12:34.633: gtk_tree_model_get_column_type: assertion 'GTK_IS_TREE_MODEL (tree_model)' failed

(lutris:5705): Gtk-CRITICAL **: 22:12:34.633: gtk_combo_box_get_active_id: assertion 'gtk_tree_model_get_column_type (model, column) == G_TYPE_STRING' failed
Running /bin/bash -c xdg-mime default modorganizer2-nxm-handler.desktop x-scheme-handler/nxm
Initial process has started with pid 5898
Game is considered started.
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 0

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 37319384 bytes (36 MiB)                                             

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/openjdk/OpenJDK8U-jre_x64_windows_8u252b09.zip
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/openjdk/OpenJDK8U-jre_x64_windows_8u252b09.zip
Type = zip
Physical Size = 37319384

Everything is Ok                             

Folders: 15
Files: 121
Size:       94859720
Compressed: 37319384

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 384875 bytes (376 KiB)                                           

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/fose/fose_v1_2_beta2.7z
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/fose/fose_v1_2_beta2.7z
Type = 7z
Physical Size = 384875
Headers Size = 3155
Method = LZMA:6m BCJ2
Solid = +
Blocks = 2

Everything is Ok

Folders: 8
Files: 184
Size:       4753751
Compressed: 384875

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 598435 bytes (585 KiB)                                           

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/f4se/f4se_0_06_21.7z
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/f4se/f4se_0_06_21.7z
Type = 7z
Physical Size = 598435
Headers Size = 5414
Method = LZMA2:22 LZMA:20 BCJ2
Solid = +
Blocks = 2

Everything is Ok

Folders: 14
Files: 393
Size:       3284040
Compressed: 598435

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 712979 bytes (697 KiB)                                           

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/nvse/nvse_5_1_beta4.7z
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/nvse/nvse_5_1_beta4.7z
Type = 7z
Physical Size = 712979
Headers Size = 4131
Method = LZMA2:6m LZMA:20 BCJ2
Solid = +
Blocks = 2

Everything is Ok

Folders: 12
Files: 231
Size:       4902927
Compressed: 712979

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 4076702 bytes (3982 KiB)                                         

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/mwse/mwse.zip
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/mwse/mwse.zip
Type = zip
Physical Size = 4076702

Everything is Ok

Folders: 23
Files: 127
Size:       14120925
Compressed: 4076702

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 1662426 bytes (1624 KiB)                                         

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/obse/obse_0021.zip
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/obse/obse_0021.zip
Type = zip
Physical Size = 1662426
Comment = ; WinRAR
; Multifunctional Integrated Archive Manager

Everything is Ok

Folders: 12
Files: 342
Size:       6092629
Compressed: 1662426
Running /bin/bash -c printf '\x90\x90\x90' | dd conv=notrunc of='/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/oblivion-script-extender/obse_loader.exe' bs=1 seek=$((0x14cb))
Initial process has started with pid 5953
Game is considered started.
3+0 enregistrements lus
3+0 enregistrements écrits
3 bytes copied, 2,6499e-05 s, 113 kB/s
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 0

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 624845 bytes (611 KiB)                                           

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/skse/skse_1_07_03.7z
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/skse/skse_1_07_03.7z
Type = 7z
Physical Size = 624845
Headers Size = 8072
Method = LZMA2:22 LZMA:20 BCJ2
Solid = +
Blocks = 2

Everything is Ok

Folders: 14
Files: 544
Size:       3591799
Compressed: 624845

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)

Scanning the drive for archives:
1 file, 755625 bytes (738 KiB)                                             

Extracting archive: /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/skse64/skse64_2_00_17.7z
--
Path = /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/skse64/skse64_2_00_17.7z
Type = 7z
Physical Size = 755625
Headers Size = 7057
Method = LZMA2:6m LZMA:20 BCJ2
Solid = +
Blocks = 2

Everything is Ok

Folders: 13
Files: 520
Size:       4426059
Compressed: 755625
Running /bin/bash -c # clear envs set by Lutris
export LD_LIBRARY_PATH=""

nexus_game_id=skyrimspecialedition
runner=proton

if [ ! -f "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/gamesinfo/$nexus_game_id.sh" ]; then
    "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" errorbox \
        "Could not find gameinfo for '$nexus_game_id', remove Lutris cache and try again"
    exit 1
fi

source "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/gamesinfo/$nexus_game_id.sh"

if [ -z "$game_appid" ]; then
    echo "ERROR: empty game_appid" >&2
    exit 1
elif [ -z "$game_steam_subdirectory" ]; then
    echo "ERROR: empty steam_subdirectory" >&2
    exit 1
fi

shared="/home/emmanueldev/.local/share/modorganizer2"

case "$runner" in
    proton)
        mo2_tricks="vcrun2019"
        mo2_options=""

        if [ -z "$steam_library" ]; then
            steam_library=$("/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/find-library-for-appid.sh" $game_appid)
        fi

        if [ ! -d "$steam_library" ]; then
            "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" errorbox \
                "Could not find '$game_steam_subdirectory' in your Steam library"
            exit 1
        fi

        game_prefix="$steam_library/steamapps/compatdata/$game_appid/pfx"
        game_installation="$steam_library/steamapps/common/$game_steam_subdirectory"
        game_tricks="$game_protontricks"

        echo -e \
        "#!/bin/bash\n\n'$shared/proton-launcher.sh' $mo2_options $game_proton_options \"\$@\" $game_appid '/home/emmanueldev/Games/mod-organizer-2/ModOrganizer2/ModOrganizer.exe'" \
        > "/home/emmanueldev/Games/mod-organizer-2/run.sh"

        echo -e \
        "#!/bin/bash\n\n'$shared/proton-launcher.sh' $mo2_options $game_proton_options $game_appid '/home/emmanueldev/Games/mod-organizer-2/ModOrganizer2/nxmhandler.exe' \"\$1\"" \
        > "/home/emmanueldev/Games/mod-organizer-2/download.sh"
        ;;

    wine)
        mo2_tricks="vcrun2019 dotnet40"
        mo2_options="--proton-wine --winever 5.*"

        "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" warnbox \
            "As of version 5.7, Wine still does not fully support Mod Organizer 2.2.1 and later.\nFor this reason this installer uses the Wine version bundled with Proton 5.0.\nMake sure you have Steam and Proton 5.0 installed on your system"

        game_prefix=$( \
            "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" directorypicker \
                "Inform the path to your wineprefix" \
        )
        if [ -z "$game_prefix" ]; then
            echo "ERROR: Installation canceled by user" >&2
            exit 1
        fi

        game_tricks="$game_winetricks"

        install_path_candidates=( \
            "$game_prefix/drive_c/Program Files (x86)/Steam/steamapps/common/$game_steam_subdirectory" \
            "$game_prefix/drive_c/GOG Games/$game_steam_subdirectory" \
        )
        if [ -n "$game_gog_subdirectory" ]; then
            install_path_candidates+=( \
                "$game_prefix/drive_c/GOG Games/$game_gog_subdirectory" \
            )
        fi
        if [ "$nexus_game_id" == "fallout3" ]; then
            install_path_candidates+=( \
                "$game_prefix/drive_c/Program Files (x86)/Steam/steamapps/common/Fallout 3" \
                "$game_prefix/drive_c/GOG Games/Fallout 3" \
            )
        fi

        for path in "${install_path_candidates[@]}"; do
            echo "Searching for game at '$path'"
            if [ -d "$path" ]; then
                echo "Found game"
                game_installation="$path"
                break
            fi
        done

        if [ -z "$game_installation" ]; then
            game_installation=$( \
                "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" directorypicker \
                    "Could not automatically find the game.\nPlease inform the path where the game is installed" \
            )
            if [ -z "$game_installation" ]; then
                echo "ERROR: installation canceled by user" >&2
                exit 1
            fi
        fi

        # TODO: uncomment once Wine fixes issues with VC2019
        # winever=$( \
        #     "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" textentry \
        #         "Inform which version of Wine you're using\nLeave as * to use the latest version available on Lutris" \
        #         "*" \
        # )
        # if [ -z "$winever" ]; then
        #     echo "ERROR: Installation canceled by user" >&2
        #     exit 1
        # fi

        echo -e \
        "#!/bin/bash\n\nWINEPREFIX='$game_prefix' '$shared/wine-launcher.sh' $mo2_options $game_wine_options \"\$@\" '/home/emmanueldev/Games/mod-organizer-2/ModOrganizer2/ModOrganizer.exe'" \
        > "/home/emmanueldev/Games/mod-organizer-2/run.sh"

        echo -e \
        "#!/bin/bash\n\nWINEPREFIX='$game_prefix' '$shared/wine-launcher.sh' $mo2_options $game_wine_options '/home/emmanueldev/Games/mod-organizer-2/ModOrganizer2/nxmhandler.exe' \"\$1\"" \
        > "/home/emmanueldev/Games/mod-organizer-2/download.sh"
        ;;
esac

if [ -d "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/${nexus_game_id}-script-extender" ]; then
    echo "Installing script extender..."

    output=$( \
        cp -af \
        "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/${nexus_game_id}-script-extender/." \
        "$game_installation/" 2>&1 \
    )
    if [ "$?" != "0" ]; then
        "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" errorbox \
            "Error while installing script extender: $output"
        exit 1
    fi
fi

mkdir -p "$game_prefix/drive_c/java"
output=$( \
    cp -af \
    "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/extracted-openjdk/." \
    "$game_prefix/drive_c/java/" 2>&1 \
)
if [ "$?" != "0" ]; then
    "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" errorbox \
        "Error while installing OpenJDK: $output"
    exit 1
fi

WINEPREFIX="$game_prefix" \
"/home/emmanueldev/.local/share/lutris/runtime/winetricks/winetricks" -q $mo2_tricks $game_tricks
if [ "$?" != "0" ]; then
    "/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/utils/dialog.sh" errorbox \
        "Error while installing winetricks, please run Lutris from a terminal and check the logs"
    exit 1
fi

# workaround to stop installation if command fails
echo "success" > '/home/emmanueldev/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation'
Initial process has started with pid 5978
Game is considered started.
Searching for Steam in '/home/emmanueldev/.steam'
Found Steam
Searching for game in library '/home/emmanueldev/.steam/steam'
Searching for game in library '/home/emmanueldev/SteamLibrary'
Found game
Installing script extender...
------------------------------------------------------
wineserver not found!
------------------------------------------------------
Gtk-Message: 22:12:38.348: GtkDialog mapped without a transient parent. This is discouraged.
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 256
2020-06-28 22:13:21,834: I can't move /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
2020-06-28 22:13:21,835: Error while completing task <bound method CommandsMixin.move of <lutris.installer.interpreter.ScriptInterpreter object at 0x7f8f428a1b38>>: I can't move /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
<class 'lutris.installer.errors.ScriptingError'> I can't move /home/emmanueldev/.cache/lutris/installer/mod-organizer-2/successful-prefix-preparation, it does not exist
None
  File "/usr/lib/python3/dist-packages/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/lutris/installer/commands.py", line 286, in move
    raise ScriptingError("I can't move %s, it does not exist" % src)
Gtk-Message: 22:13:54.455: GtkDialog mapped without a transient parent. This is discouraged.

Is anyone able to help me out there?
Thank you in advance for your help.

**Comment by EmmanuelD69** [comment]
Update 29th June 2020: 
I finally achieved to install MO2 in 2 steps:
1-removing / re-installing Winetricks
2-using solution that https://github.com/rockerbacon/lutris-skyrimse-installers/issues/61.

Now i am facing another issue, this time when starting MO2, i got a error alert window popup with message:
Wine C++ Runtime Library
Runtime Error!
Trying to figure out what's wrong now....

**Comment by EmmanuelD69** [comment]
Here is part of the log when i am starting MO2 and create a new instance for SkyrimSE, can anyone help me out?
Thank you

Proton: Upgrading prefix from 5.9-GE-2-MF to 5.0-1 (/home/emmanueldev/SteamLibrary/steamapps/compatdata/489830/)
Proton: Prefix has an invalid version?! You may want to back up user files and delete this prefix.
esync: up and running.
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
wine: configuration in L"/home/emmanueldev/SteamLibrary/steamapps/compatdata/489830/pfx" has been updated.
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
Setting breakpad minidump AppID = 489830
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561198237534392 [API loaded no]
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
wine: Unhandled exception 0x80000003 in thread d0 at address 000000007BCD4688 (thread 00d0), starting debugger...
0x000000007bcd4688 DbgBreakPoint in ntdll: int	$3

**Comment by EmmanuelD69** [comment]
Update June 29th 2020 part2:
Finally i got MO2 installed and running!! 
To make it short,
1-I deleted the folder into : /home/emmanueldev/SteamLibrary/steamapps/compatdata/
2-Deleted the broken version of MO2 on lutris
3-restarted Steam and made sure that my version of Steam play Proton is the latest one i uploaded.
4-Launched a new session of SkyrimSE via Lutris which recreated a folder into /home/emmanueldev/SteamLibrary/steamapps/compatdata/ with the correct version file of Proton.
5-Installed again MO2 using shell command: "lutris -i "$HOME/Downloads/modorganizer2.yml""

MO2 installed properly this time and i could set it all up! 
Hope this will help anyone that goes through the damn long process i had to get Skyrim SE to work.
Enjoy

**Comment by rockerbacon** [comment]
> Proton: Upgrading prefix from 5.9-GE-2-MF to 5.0-1 (/home/emmanueldev/SteamLibrary/steamapps/compatdata/489830/)

This was likely the problem. It seems Steam created a prefix for a custom Proton build at some point.

Closing this issue since the user found a solution.
